sphinx_needs/lsp/esbonio.py

```diff
@@ -19,10 +19,33 @@
     Range,
     TextEdit,
 )
+from sphinx.application import Sphinx
 
 from sphinx_needs.lsp.needs_store import NeedsStore
 
 
+def get_needs_json(app: Sphinx) -> Optional[Path]:
+    """
+    Get the location of needs.json.
+    """
+
+    needs_json = None
+    # check if needs.json is built automatically during each sphinx-build,
+    # which requres needs_build_json = True in conf.py
+    outdir = Path(app.outdir)
+    if hasattr(app.config, "needs_build_json") and app.config.needs_build_json:
```

Comment:
does this work instead?

```suggestion
    if getattr(app.config, "needs_build_json", False):
```

Reply:
I like this getattr(), it is much cleaner.